llvm/tools/llvm-symbolizer/llvm-symbolizer.cpp

```diff
@@ -337,6 +337,13 @@ static void symbolizeInput(const opt::InputArgList &Args,
   object::BuildID BuildID(IncomingBuildID.begin(), IncomingBuildID.end());
   uint64_t Offset = 0;
   StringRef Symbol;
+
+  // Empty input string may be used to check if the process is alive. Do not
```

Comment:
```suggestion
  // An empty input string may be used to check if the process is alive and responding to input. Do not
```
(and reflow the comment as necessary)